README.md

```diff
@@ -122,6 +122,17 @@ the `dm-ssp-jenkins` user. The Client ID and Client Secret must be stored in the
 *jenkins-vars/jenkins.yaml*, and must be stored as a nested dict under `jenkins_github_auth_by_hostname`. This allows
 us to maintain multiple Jenkins instances (if required). These credentials are deployed via the `config` task tag.
 
+## Creating a new Jenkins instance
```

Comment:
Is this all the info that was needed to create a Jenkins instance? You didn't need to follow any of the stuff in the manual?

Reply:
Oh of course... I was going to link to that manual page here and then completely forgot

Reply:
Actually, I'm just going to replace what I wrote here with a link to the manual. It's got far more detail and there's no point duplicating the documentation